src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.LoggerFactoryWrapper.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+internal partial class RazorLanguageServer
+{
+    private class LoggerFactoryWrapper : ILoggerFactory, IRazorLoggerFactory
+    {
+        private IRazorLoggerFactory _loggerFactory;
+
+        public LoggerFactoryWrapper(IRazorLoggerFactory loggerFactory)
```

Comment:
Please consider renaming to `razorLoggerFactory`. it makes the implementation here a bit more clear and concise.